lib/svg-sprite/shape.js

```diff
@@ -369,7 +369,7 @@ SVGShape.prototype._initSVG = function() {
     this.dom = new DOMParser({
         locator: {},
         errorHandler(level, message) {
-            const e = new Error(format('Invalid SVG file (%s)', message.split('\n').join(' ')));
+            const e = new Error(format('Invalid SVG file (%s)', message.replace(/\s+/g, ' ').split('\n').join(' ')));
```

Comment:
the `message` contains multiple spaces here. It appears normally in console, but it's hard to test.

Reply:
I'm not a fan of changing the code just for the tests. I wouldn't test this for the time being if it's causing issues.